packages/discord.js/typings/index.d.ts

```diff
@@ -586,9 +586,6 @@ export abstract class CommandInteraction<Cached extends CacheType = CacheType> e
     option: APIApplicationCommandOption,
     resolved: APIApplicationCommandInteractionData['resolved'],
   ): CommandInteractionOption<Cached>;
-  private transformResolved(
```

Comment:
lets export this method still, as it was technically exported before

Reply:
Resolved with 420288d90e94f218f780323bd85930d20b6132e3.